private-templates-service/client/src/Welcome.tsx

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
-import {
-  Button,
-  Jumbotron } from 'reactstrap';
+import React from "react";
+import { Button, Jumbotron } from "reactstrap";
 
-function WelcomeContent(props) {
+interface Props {
```

Comment:
I'm seeing this structure a lot, it might be worth creating its own type to contain this data if you dont use a global store

Reply:
Will be a part of store.